perun-core/src/test/java/cz/metacentrum/perun/core/entry/GroupsManagerEntryIntegrationTest.java

```diff
@@ -1450,6 +1451,61 @@ public void convertGroupsToRichGroupsWithAttributesWithListOfNamesTest() throws
 		assertEquals("Both lists should be same", Arrays.asList(richGroup1, richGroup2), groupsManagerBl.convertGroupsToRichGroupsWithAttributes(sess, Arrays.asList(group, group2), attrNames));
 	}
 
+	@Test(expected = ExternalyManagedException.class)
+	public void addMemberToSynchronizedGroup() throws Exception {
+		System.out.println(CLASS_NAME + "addMemberToSynchronizedGroup");
+
+		vo = setUpVo();
+		setUpGroup(vo);
+
+		Attribute synchroAttr = new Attribute(perun.getAttributesManager().getAttributeDefinition(sess, "urn:perun:group:attribute-def:def:synchronizationEnabled"));
+		synchroAttr.setValue("true");
+		perun.getAttributesManager().setAttribute(sess, group, synchroAttr);
+
+		Member member = setUpMember(vo);
+		groupsManager.addMember(sess, group, member);
+	}
+	
```

Comment:
Hi, there should be a ``@Test`` annotation above the method, otherwise it's not started during testing.